newIDE/app/src/UI/SemiControlledAutoComplete.js

```diff
@@ -131,227 +79,200 @@ type Props = {|
+      )
+        optionList.shift();
+
+      return optionList;
+    };
+
+    const helperText = this.props.helperMarkdownText ? (
+      <MarkdownText source={this.props.helperMarkdownText} />
     ) : null;
 
+    const getOptionLabel = (option: Option): string => {
+      if (option.value) return option.value;
+      return getCurrentInputValue();
+    };
+
+    const handleChange = (event, option: Option | string): void => {
+      if (option !== null && option.type !== 'separator') {
+        if (typeof option === 'string') {
+          this.props.onChange(option);
+        } else {
+          if (option.onClick) option.onClick();
+          else {
+            if (this.props.onChoose) this.props.onChoose(option.value);
+            else this.props.onChange(option.value);
+          }
+        }
+      }
+      if (
+        typeof event.key === 'string' &&
+        event.key === 'Enter' &&
+        this._input.current !== null
+      )
+        this._input.current.blur();
+      //if (this.props.onRequestClose) this.props.onRequestClose();
+    };
+
+    const handleInputChange = (e, value: string, reason: string): void =>
+      this.setState({ inputValue: value });
+
+    const getDefaultStylingProps = (params: Object): Object => {
+      const { InputProps, inputProps, InputLabelProps, ...other } = params;
+      return {
+        ...other,
+        InputProps: {
+          ...InputProps,
+          className: null,
+          endAdornment: null,
+        },
+        inputProps: {
+          ...inputProps,
+          className: null,
+          onKeyDown: (event): void => {
+            // if (event.key === 'Escape' && this.props.onRequestClose)
+            //   this.props.onRequestClose();
+            if (event.key === 'Enter')
+              handleChange(event, getCurrentInputValue());
+          },
+        },
+      };
+    };
+
     return (
       <I18n>
         {({ i18n }) => (
-          <Downshift
-            inputValue={currentInputValue}
-            onChange={selectedItem => {
-              if (selectedItem !== null) {
-                if (selectedItem.onClick) {
-                  selectedItem.onClick();
-
-                  // Reset the value shown to the current value,
-                  // as the menu item clicked is not a "value" to be
-                  // chosen.
-                  this.setState({
-                    inputValue: props.value,
-                  });
-                } else {
-                  // Call onChoose, if available, as this is a real selection
-                  // of an item (contrary to onChange, which can be called even
-                  // with a partial input when the field is blurred)
-                  if (props.onChoose) props.onChoose(selectedItem.value);
-                  else props.onChange(selectedItem.value);
-                }
-              }
+          <Autocomplete
+            freeSolo
+            blurOnSelect
+            classes={{
```

Comment:
Can you just do `classes={classes}`? Does it work?

Reply:
I'd check if it works. (I guess it should)